### ISSUE TITLE
Implement and test Conformer in this repo

### ISSUE THREAD

**Issue by kordc** [SebChw/Actually-Robust-Training#17]
Look at the paper and existing repository (preferred the same as in #15), and implement it using PyTorch Lightning (2.0 preferred). Test on the same datasets as in #15

**Comment by kordc** [comment]
might be impossible given our resources 
``` A lot of training data in your required language   
(preferable >100h for fine-tuning and >1000h for new languages)

**Comment by kordc** [comment]
we can just use https://catalog.ngc.nvidia.com/models?filters=&orderBy=scoreDESC&query=label:%22Conformer-CTC%22

**Comment by kordc** [comment]
As in #15 - we can use the small or medium model. 
NVIDIA took thousands of hours of English speech, so definitely we don't have enough computing power to train a such model. 

Instead, we can try re-training the model starting from predefined weights on SLU datasets such as Snips-Smart{Lights/Speaker} or Fluent Command. Then after retraining, it'd be probably the best model we could work on later on FSTs to get the understanding part.
